homeassistant/components/telegram_bot/__init__.py

```diff
@@ -163,6 +164,10 @@
     }
 )
 
+SERVICE_SCHEMA_SEND_STICKER = SERVICE_SCHEMA_SEND_FILE.extend(
+    {vol.Optional(ATTR_STICKER_ID): cv.string}
```

Comment:
This should be required. 
```suggestion
    {vol.Required(ATTR_STICKER_ID): cv.string}
```

Reply:
Ah I see now that you're extending it.